components/molecules/CardContainerWithRows/CardContainerWithRows.js

```diff
@@ -0,0 +1,120 @@
+/** @module Ranking */
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
```

Comment:
```suggestion
import React, {Component, Fragment} from 'react';
```

So that you can just do `Fragment` instead of `React.Fragment`.

Reply:
Updated to use just Fragment in https://github.com/UnitedIncome/components/pull/867/commits/21c1882106753193b3b995f2e26d6b5578cca960